packages/components/skeleton/src/skeleton-item.vue

```diff
@@ -1,26 +1,21 @@
 <template>
   <div :class="[ns.e('item'), ns.e(variant)]">
-    <img-placeholder v-if="variant === 'image'" />
+    <ImgPlaceholder v-if="variant === 'image'" />
```

Comment:
I think we should preserve the old way by using `kebab-case` for components since it is not a conflict name against the DOM API